packages/docusaurus-plugin-content-blog/src/__tests__/blogUtils.test.ts

```diff
@@ -295,3 +297,49 @@ describe('linkify', () => {
     } as BlogBrokenMarkdownLink);
   });
 });
+
+const defaultValuesForOtherKeys: Omit<BlogPostMetadata, 'date'> = {
```

Comment:
weird variable name 😅 why not put this directly inside `createBlogPost` and let it accept Partial metadata too?


```js
import type {DeepPartial} from 'utility-types';

const createBlogPost = (args: DeepPartial<BlogPost>): BlogPost
```